recipes/datefinder/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "datefinder" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/akoumjian/datefinder/archive/v0.7.1.tar.gz"
+  sha256: acab2bebfdb09fdc9e9d512b280bf676a9a03f286c666234d50cfe65a5da7f7d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
```

Comment:
You need to add `pip` to your host requirements if you're going to use `pip` above to build the package.

Reply:
Done